### ISSUE TITLE
Conversion errors.

### ISSUE THREAD

**Issue by pierreSCpapon** [uPesy/easyeda2kicad.py#2]
First off, thank you for this amazing tool, it really speeds up development whilst using LCSC as part supplier.

I wanted to signal some import errors I have noticed:

1) Rotation of pad is wrong on converted footprint
Part in question:
https://lcsc.com/product-detail/Low-Dropout-Regulators-LDO_UMW-Youtai-Semiconductor-Co-Ltd-XC6206P332PR_C545581.html

EasyEDA preview:
![image](https://user-images.githubusercontent.com/78420302/153804574-64d1f403-3720-4cbc-aee5-1af60936d45d.png)

Converted footprint:
![image](https://user-images.githubusercontent.com/78420302/153804665-5fc0a93c-0006-4838-bef5-24fa6cad2d23.png)

2) Symbol shape missing (seems to be triangles)
Part in question:
https://lcsc.com/product-detail/Schottky-Barrier-Diodes-SBD_DIODES_1N5819HW-7-F_1N5819HW-7-F_C82544.html

EasyEDA preview:
![image](https://user-images.githubusercontent.com/78420302/153805102-82c079b4-f8c4-45b6-8a00-ce15af024be6.png)

Converted symbol:
![image](https://user-images.githubusercontent.com/78420302/153805151-8cb2e3d0-5753-4c6c-8e40-32506688bd24.png)

more examples:
https://lcsc.com/product-detail/Light-Emitting-Diodes-LED_FM-B2020RGBA-HG_C108793.html
![image](https://user-images.githubusercontent.com/78420302/153805221-f39ba7de-f306-4345-8177-5b57f5c5a214.png)
![image](https://user-images.githubusercontent.com/78420302/153805280-337d5708-ac34-4e3d-a40d-5e643e84d2e1.png)

https://lcsc.com/product-detail/Others_HYP-1TS009C-2600-5000_C319427.html
![image](https://user-images.githubusercontent.com/78420302/153805409-15e77594-8d7b-41c1-b9e1-2e32570a64a4.png)
![image](https://user-images.githubusercontent.com/78420302/153805435-726315ac-aea3-4aa6-a802-0ad7433e3204.png)

Let me know if you need more info or would like me to test anything, thank you.

**Comment by uPesy** [comment]
@pierreSCpapon Thanks for your report.
I have also these issues when testing.
For the footprint, there is a rotation and an offset issue. Do you have notice this behaviour with others components when there is an uncommon pad ?
For the symbol, i have noticed that EasyEda add 2 more types : Polygon and path.
You should have warning when you run the script right ?
```
[*] Creating Kicad symbol library for LCSC id : C108793
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
        [-] Unknow symbol designator : PG
[*] Creating Kicad footprint library for LCSC id : C108793
```
I will try to implement that this week-end.
Thanks you again to point out issue on some components since i can't test them all :)

**Comment by uPesy** [comment]
I have fix the issues for symbol.
Feel free to report other bugs. I will fix the footprint issue later.

**Comment by pierreSCpapon** [comment]
Awesome! Thank you.

I have actually just tested the new version and found an offset error with the symbol.

https://lcsc.com/product-detail/Buzzers_S-S-SFN-17075PA10_C360608.html

**Comment by uPesy** [comment]
All the symbols issues mentionned previously should be fixed now in the new release #18 
You can update the symbol by using `--overwrite` argument 
```
easyeda2kicad --symbol --footprint --lcsc_id=C319427  --overwrite
```
